File: lc-blind75/ContainsDuplicate(lc-217)/Bruteforce/FindDuplicate.py
# tc:O(n^2) sc:O(1)
def containsDuplicate(nums):
    isDuplicate=False
    n=len(nums)
    for i in range(n):
        for j in range(i+1,n):
            if nums[i]==nums[j]:
                isDuplicate=True
                break
    return isDuplicate

n=int(input())
nums=list(map(int,input().split()))
print(containsDuplicate(nums))

File: lc-blind75/ContainsDuplicate(lc-217)/Bruteforce/FindDuplicate.java
//tc:O(n^2) sc:O(1)
import java.util.*;
public class FindDuplicate {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int nums[]=new int [n];
        for(int i=0;i<n;i++){
            nums[i]=sc.nextInt();
        }
        System.out.println(duplicate(nums));
        sc.close();
    }
    public static boolean duplicate(int nums[]){
        int n=nums.length;
        boolean isDuplicate=false;
        for(int i=0;i<n;i++){
            for(int j=i+1;j<n;j++){
                if(nums[i]==nums[j]){
                    isDuplicate=true;
                    break;
                }
            }
        }
        return isDuplicate;
    }
}
